capstone/src/main/webapp/js/servletConnection.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Gets suggestions to display to the users
+ * @return{void}
+ */
+function getSuggestions() {
+  fetch('/search').then((response) => response.json()).then((suggestions) => {
+    provideSuggestions(suggestions);
```

Comment:
```suggestion
  fetch('/search').then((response) => response.json())
      .then(provideSuggestions);
```
Since it's expecting a function object, you might as well provide the function directly instead of wrapping it in an arrow function.